examples/aggregator/module/rust/src/lib.rs

```diff
@@ -75,24 +77,22 @@ impl AggregatorNode {
         Ok(())
     }
 
-    fn send_aggregated_data(&self, bucket: String, svec: SparseVector) {
+    fn report(&self, bucket: String, svec: SparseVector) {
```

Comment:
Just curious, why is this not returning a `Result` too? The callers already handle logging of errors (line 114).